src/with_clause/alter_table_with_clause.h

```diff
@@ -15,7 +15,8 @@
 
 typedef enum AlterTableFlags
 {
-	AlterTableFlagCompressEnabled = 0,
+	AlterTableFlagChunkTimeInterval = 0,
```

Comment:
Why do we have ` = 0` by the way? Looks as if we cared about the absolute values, but we don't.

Reply:
since those are used as array index we want them to start with 0 to align with array size

Reply:
but those values are never persisted so it's not required that they stay in same order